tests/test_local_json.py

```diff
@@ -135,6 +135,19 @@ def test_graph():
     assert s1.pred == 'hasRelatedSynonym'
     assert s1.xrefs == ['GOC:mah']
 
+    GOSLIM = 'goslim_generic'
+    subsets = ont.subsets(NUCLEUS)
+    print("SUBSETS: {}".format(subsets))
+    assert GOSLIM in subsets
```

Comment:
+1 for a unit test framework, we use plain old python unittest in the dipper code:
https://docs.python.org/3/library/unittest.html

The future of nose is unclear, from their docs:
Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using Nose2, py.test, or just plain unittest/unittest2.